projects/rero/ng-core/src/lib/record/editor/editor.component.ts

```diff
@@ -168,6 +169,9 @@ export class EditorComponent implements OnInit, OnChanges, OnDestroy {
         this.setSchema(this.schema);
       }
     }
+    if (changes.schema && changes.schema.isFirstChange()) { // Schema has changed
+      this.setSchema(changes.schema.currentValue);
+    }
```

Comment:
Can be referenced in commit message

Reply:
It is actually as the schema has been added to inputs.